gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,22 +1,3 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is not related to Picocli... Since license-related changes are sensitive, would it make sense to make this change in a separate PR?

Reply:
Yes, my bad, let me isolate this change in another PR.